pyro/distributions/transforms/affine_coupling.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import absolute_import, division, print_function
+
+import torch
+from torch.distributions import constraints
+
+from pyro.distributions.torch_transform import TransformModule
+from pyro.distributions.util import copy_docs_from
+
+
+def clamp_preserve_gradients(x, min, max):
```

Comment:
if this is being reused in several places it should be put in a utils file somewhere

Reply:
Done!